cptv-reprocess.py

```diff
@@ -66,7 +66,8 @@ def recording_range(range_s):
         raise argparse.ArgumentTypeError(
             "multiple ':', exepected format of start:end or id,id2,... "
         )
-    elif len(id_range) == 2:
+
+    if len(id_range) == 2:
```

Comment:
why change from elif?

Reply:
because the previous `if` only raises so the `elif` is unnecessary (pylint complains about it)